webapp/src/services/api/study.ts

```diff
@@ -329,17 +329,18 @@ export const mapLaunchJobDTO = (j: LaunchJobDTO): LaunchJob => ({
 });
 
 export const getStudyJobs = async (
-  sid?: string,
+  studyId?: string,
   filterOrphans = true,
   latest = false,
 ): Promise<LaunchJob[]> => {
-  let query = sid
-    ? `?study=${sid}&filter_orphans=${filterOrphans}`
-    : `?filter_orphans=${filterOrphans}`;
-  if (latest) {
-    query += "&latest=100";
-  }
-  const res = await client.get(`/v1/launcher/jobs${query}`);
+  const queryParams = new URLSearchParams({
```

Comment:
```javascript
const queryParams = new URLSearchParams([
  ["filter_orphans", filterOrphans],
  studyId && ["study", studyId],
  latest && ["latest", 100],
].filter(Boolean));
```

Reply:
Thanks but it causes TS issues, I let you see

Reply:
I think it would be this instead:
```javascript
const queryParams = new URLSearchParams(
  [
    ["filter_orphans", String(filterOrphans)],
    ...(studyId ? [["study", studyId]] : []),
    ...(latest ? [["latest", "100"]] : []),
  ]
);
```

Reply:
```javascript
const queryParams = new URLSearchParams(
    R.filter(Boolean, {
      filter_orphans: filterOrphans,
      study: studyId,
      latest,
    }),
  );
```